lib/models/user/pushNotifications.js

```diff
@@ -1,36 +1,52 @@
 "use strict";
 
 const config  = require("../../../config.js");
-const Client = require("node-rest-client").Client;
-const client = new Client();
+const request = require("request");
 
 module.exports = function (UserSchema) {
 
     UserSchema.methods.sendPushNotification = function (payload) {
-      if (!config.pushNotificationsEnabled) return;
-      var user = this;
-
-      const username = user.email;
-      const pushData = [
-        Object.assign({username, namespace: "OrangeMobile"}, payload)
-      ];
-
-      const authArgs = {
-          headers: {"Content-Type": "application/json"},
-          data: {
-            username: config.pushNotificationsServiceUserUsername,
-            password: config.pushNotificationsServiceUserPassword
-          }
-      };
-
-      client.post(`${config.authServiceAPI}/auth/login`, authArgs, function (data, response) {
-          const { token } = data;
-          const pushNotificationArgs = {
-              headers: {"Content-Type": "application/json", "Authorization":"Bearer " + token},
-              data: {pushData}
-          };
-          client.post(`${config.notificationServiceAPI}/notifications/sendPushNotifications`, pushNotificationArgs, function (data, response) {
-          });
-      });
+        if (!config.pushNotificationsEnabled) return;
+        var user = this;
+
+        const username = user.email;
+        const pushData = [
+            Object.assign({username, namespace: "OrangeMobile"}, payload)
+        ];
+
+        const authArgs = {
+            url: `${config.authServiceAPI}/auth/login`,
+            headers: { "Content-Type": "application/json" },
+            json: true,
+            body: {
+              username: config.pushNotificationsServiceUserUsername,
+              password: config.pushNotificationsServiceUserPassword
+            }
+        };
+
+        request.post(authArgs, function (err, res) {
+            if (err) {
+                console.log(`${new Date()} Error: pushNotifications.js: Push notificatios cron job call to Auth Service failed. Aborting.`, err);
```

Comment:
Is using the winston logger an option here?

Reply:
Thanks for the feedback! The amida winston logger would be great for this, but it's not implemented in this repo yet. This repo only uses the express winston express middleware, which, AFAIK, isn't for this purpose. Before implementing, I did a quick check, and it looks like `console.log` is the standard for this repo still.